### PR TITLE
Add Product: ID.me (online identity verification service)

### PR DESCRIPTION
**Pull request by opile8**
**Type of pull request:** product addition / product edit / bug fix / other...
product addition: ID.me
- included icon and toml, used other products and rubric as guides

**Related issues:** list the related issues here (if applicable)

---

Provide more information about your pull request here. For help, see our contribution guide: http://privacyspy.org/contribute/

### CODE REVIEW

File: products/id-me.toml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  "- procedures for termination of access to Personal Information designed to curtail access to the information by terminated personnel or when there is no longer a business need for access;", 
+  "- personnel security controls designed to reduce the risk of human error, theft, fraud or misuse of facilities; and", 
+  "- physical and environmental security procedures designed to prevent unauthorized access, damage or interference to business premises and information.", 
+  "", 
+  "[Security Page]", 
+  "ID.me has been designed to comply with rigorous information security regulations including AICPA SOC 2, ISO 27001, FedRAMP, and multiple NIST 800 guidelines. Multiple ID.me clients have completed extensive technical due diligence with regard to the processing environment. The customers that have completed this due diligence are: USAA, Veterans Affairs, IRS, SSA, and Allscripts. ID.me implements role based access management, separation of duties, and multifactor authentication. Data at rest and in transit is encrypted using approved FIPS 140-2 algorithms. Personally Identifiable Information (PII) is encrypted using a rolling key and the AES-256-CBC algorithms."
```
Comment (doamatto) [inline products/id-me.toml]: This is the first I'm hearing that USAA FSB started auditing other companies /s

In seriousness, although knowing the U.S. govt and (iirc) the second biggest military-exclusive financial institution trusts them, we could probably omit that portion in the name of brevity (can leave in FIPS, FedRAMP, et al.).

File: products/id-me.toml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  "If you connect to us through a social media platform or navigate to a social media platform from one of our sites, the social media platform will collect your information separately from us. You should review the social media platforms' privacy policies to understand how they are using your information and your rights in relation to such information.", 
+  "Information We Derive", 
+  "We may derive additional information or draw inferences about you based on the information we have collected from you directly, passively, or through third parties."
+]
+notes = [
+  "ID.me previously (until 3/14/2022) allowed login to Facebook using ID.me as the sign-on service (via Facebook Connect). The privacy policy at that time included language about ID.me's collection and storage of data about those contacts (depending on the users' privacy settings). ID.me still allows a customer to [use social media accounts to sign into ID.me](https://help.id.me/hc/en-us/articles/360057107014-Connecting-social-or-third-party-accounts-to-your-ID-me-account) from accounts like Apple, Facebook, Google, or LinkedIn, but ID.me no no longer seems policy no longer mentions these by name."
```
Comment (doamatto) [inline products/id-me.toml]: Typo in this section ("but ID.me no no longer"). Retrospective might not be needed, but I'm fine with leaving it in

File: products/id-me.toml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+]
+
+[rubric.third-party-collection]
+value = "critical-only"
+citations = [
+  "Information from our partners. We acquire information from other trusted sources. These business partners might include companies, such as your mobile phone carriers, certain government agencies, licensing bodies, etc. We may also collect information about you from other sources, including service providers, data licensors and aggregators, marketing companies, programming distributors, and public databases.", 
+  "Information you provide through social media", 
+  "If you connect to us through a social media platform or navigate to a social media platform from one of our sites, the social media platform will collect your information separately from us. You should review the social media platforms' privacy policies to understand how they are using your information and your rights in relation to such information.", 
+  "Information We Derive", 
+  "We may derive additional information or draw inferences about you based on the information we have collected from you directly, passively, or through third parties."
```
Comment (doamatto) [inline products/id-me.toml:47]: I think the vagueness of this statement makes `critical-only` hard to justify. From the experiences I've heard others have, it's to check against American credit bureaus for identity verification, but the wording would have to limit it to just that and/or whatever else they collect from.

File: products/id-me.toml [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+]
+notes = [
+  "Some information you provide to ID.me may be retained for up to 36 months for legal compliance purposes following a deletion request. Biometric information will automatically \"age off\" after 36 months, if not sooner."
+]
+
+[rubric.data-breaches]
+value = "no"
+notes = [
+  "Policy makes no mention of data breach procedures or notification to potential victims.", 
+  "It does, however, admit \"[...] that no data transmission or storage can be guaranteed to be 100% secure.\""
```
Comment (doamatto) [inline products/id-me.toml]: Second bit isn't necessary — most services have this line

File: products/id-me.toml [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+]
+
+[rubric.third-party-access]
+value = "yes-specified-critical"
+citations = [
+  "We may share your Personal Information with entities necessary to validate your ID.me Account and provide our Services to you. In order to verify your identity and eligibility to receive discounts and other benefits from our partners and other service providers, we may provide your Personal Information to third parties such as government agencies, telecommunications networks, financial institutions or other trusted and reliable sources of information. Our provision of your Personal Information to the foregoing parties is solely to verify your identity and eligibility for ID.me Services. We have established relationships with Registration Authorities similar to the entities described above whereby the Personal Information you provide to us will be transmitted to them using industry standard encryption tools, designed to protect such information from unauthorized access.",
```
Comment (doamatto) [inline products/id-me.toml:89]: "Other trusted and reliable sources of information" is too vague for this to be `yes-specified-critical`. Based on the policy, `yes-unspecified-critical` would be OK

File: products/id-me.toml [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+]
+
+[rubric.data-collection-reasoning]
+value = "yes"
+citations = [
+  "[Privacy Policy Page]", 
+  "Verification information. When you verify yourself, either individually or as part of a community, with ID.me you provide us with Personal Information that may include your name, date of birth, social security number and/or other government issued identification numbers, copies of your government issued identification card (e.g., license or passport), email address, phone number, mailing address, and certain photographic images, and biometric data. You may also be asked to provide community affiliations (e.g., Military, First Responder, Student, Veteran, etc.), memberships, educational degrees, and professional certifications.",
```
Comment (doamatto) [inline products/id-me.toml]: It makes more sense to quote and score based on section 2 ("How We May Use Your Information and Why").

File: products/id-me.toml [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+  "[Biometric Policy Page]", 
+  "We use your Biometric Information only as follows:", 
+  "- To verify your identity when you are opening an account or using our Services;", 
+  "- To authenticate use of your account and the Services for a transaction;", 
+  "- To prevent fraudulent uses of ID.me’s Services or the creation of multiple accounts; and", 
+  "- To comply with legal obligations or comply with a request from law enforcement or government entities where not prohibited by law."
+]
+notes = [
+  "ID.me uses this information to become a trusted middleman to verify your affiliation.",
```
Comment (doamatto) [inline products/id-me.toml]: I think explaining the point of ID.me might be out of scope (for lack of better wording)

File: products/id-me.toml [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+]
+notes = [
+  "If ID.me is strictly used as identity verification for (state/federal) government services then this is on an \"opt-in basis\" so no marketing occurs (see also \"behavioral marketing\")."
+]
+
+[rubric.law-enforcement]
+value = "reasonable"
+citations = [
+  "[Privacy Policy Page]", 
+  "We may share information as required with the United States federal government and certain state governments. ID.me does not provide any government with direct and unfettered access to our user's data, and we do not provide any government with our encryption keys or the ability to break our encryption. We may share certain Personal Information associated with an ID.me account with government entities where we reasonably believe that account may be engaging in fraud.",
```
Comment (doamatto) [inline products/id-me.toml]: The section above this ("We may share information as needed in order to comply with legal processes, to protect ourselves, or improve our Services.") would also be a good idea.

File: products/id-me.toml [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+  "Our legal and compliance teams review all requests to ensure they are valid, reject those that are not valid, and only provide the data specified in the subpoena or similar court order.", 
+  "[Biometric Policy Page]", 
+  "ID.me will only share your Biometric Information with our partners in the following circumstances:", 
+  "As required with other third parties where permitted by law to enforce our Terms of Service, to comply with legal obligations, or to cooperate with law enforcement agencies concerning conduct or activity that we reasonably believe may violate federal, state, or local law when required by a subpoena, warrant, or other court ordered legal action, and to prevent harm, loss or injury to others.", 
+  "To third party service providers that perform functions on our behalf. These service providers are limited to using the Biometric Information to assist in our provision of Services, and must maintain any Biometric Information we share in a secure fashion."
+]
+
+[rubric.list-collected]
+value = "exhaustively"
+citations = [
```
Comment (doamatto) [inline products/id-me.toml:150]: They collect more than what's mentioned here. Basically the whole of section 1

File: products/id-me.toml [hunk 10]
```diff
@@ -0,0 +1,149 @@
+name = "ID.me"
+description = "Consumers can verify their identity with ID.me once and seamlessly log in across websites without having to create a new login or verify their identity again."
+slug = "id-me"
+hostnames = ["id.me"]
+sources = ["https://www.id.me/privacy", "https://id.me/biometric", "https://www.id.me/security", "https://id.me/washington-privacy"]
```
Comment (doamatto) [inline products/id-me.toml]: I don't *think* I saw anything quoted from the Washington state privacy page ? We don't need to quote from there anyways unless it's something that applies to everyone

**Review by doamatto — CHANGES_REQUESTED** [review]
A few things that need tweaking. There was also the ID.me Rx section near the bottom that I forget if was included in this. Overall, good start ^_^

File: products/id-me.toml (hunk 10 above)
Comment (opile8) [reply products/id-me.toml]: Removed. Was used as a placeholder as I explored all of the privacy policies on the site.

File: products/id-me.toml (hunk 1 above)
Comment (opile8) [reply products/id-me.toml]: Removed that sentence. Left other security measures.

File: products/id-me.toml (hunk 3 above)
Comment (opile8) [reply products/id-me.toml:47]: Changed rubric grade to `yes`.

File: products/id-me.toml (hunk 4 above)
Comment (opile8) [reply products/id-me.toml]: Removed second line of "notes"

File: products/id-me.toml (hunk 5 above)
Comment (opile8) [reply products/id-me.toml:89]: I understand the distinction in the rubric now. Changed as you suggested.

File: products/id-me.toml (hunk 6 above)
Comment (opile8) [reply products/id-me.toml]: cited section 2, included parts about "for marketing purposes"

File: products/id-me.toml (hunk 7 above)
Comment (opile8) [reply products/id-me.toml]: Updated notes to point to the "non-critical purposes" grade.

File: products/id-me.toml (hunk 8 above)
Comment (opile8) [reply products/id-me.toml]: Added citation to law-enforcement for paragraph you referenced.

File: products/id-me.toml (hunk 9 above)
Comment (opile8) [reply products/id-me.toml:150]: Added section 1

**Review by doamatto — APPROVED** [review]
Looks good now. Thanks !

**Pull request merged**